### PR TITLE
fix #299727: Change Transpose "By Key" to "To Key"

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
and add a tooltip mentioning that the key is specified at concert pitch

image

Resolves https://musescore.org/en/node/299727 and stems from a discussion in https://musescore.org/en/node/299670#comment-972012

### CODE REVIEW

File: mscore/file.cpp [hunk 1]
```diff
@@ -3422,7 +3422,7 @@ bool MuseScore::exportTransposedScoreToJSON(const QString& inFilePath, const QSt
 
       TransposeMode mode;
       const QString modeName = options["mode"].toString();
-      if (modeName == "by_key")
+      if (modeName == "by_key" || nodename == "to_key")
```
Comment (Jojo-Schmitz) [inline mscore/file.cpp]: leaving the old option for compatibility

**Comment by riaanvn** [comment]
Hi @Jojo-Schmitz 

(I unfortunately missed the initial discussion at #299727. I see now why this is more likely to get attention in the PR)

I don't necessarily see a problem with "By Key" because

a) it matches the cadence / pattern of "By Interval"
b) I see "By" as an abbreviation for "according to", whereas "To" means "To destination"

I don't see either as wrong. If the consensus is that "To Key" is better, I am happy to accept it.

**Comment by Jojo-Schmitz** [comment]
It is a direction here, in the next part of the dialog you set whether up, down or to nearest

**Comment by Jojo-Schmitz** [comment]
Possible conflicts with #5641

**Pull request merged**